tests/test_conda_env.py

```diff
@@ -0,0 +1,152 @@
+            "conda.cli.main_env_export.configure_parser",
+        ),
+        ("conda_env.cli.main_export.execute", "conda.cli.main_env_export.execute"),
+        ("conda_env.cli.main_list.description", None),
+        ("conda_env.cli.main_list.example", None),
+        (
+            "conda_env.cli.main_list.configure_parser",
+            "conda.cli.main_env_list.configure_parser",
+        ),
+        ("conda_env.cli.main_list.execute", "conda.cli.main_env_list.execute"),
+        ("conda_env.cli.main_remove._help", None),
+        ("conda_env.cli.main_remove._description", None),
+        ("conda_env.cli.main_remove._example", None),
+        (
+            "conda_env.cli.main_remove.configure_parser",
+            "conda.cli.main_env_remove.configure_parser",
+        ),
+        ("conda_env.cli.main_remove.execute", "conda.cli.main_env_remove.execute"),
+        ("conda_env.cli.main_update.description", None),
+        ("conda_env.cli.main_update.example", None),
+        (
+            "conda_env.cli.main_update.configure_parser",
+            "conda.cli.main_env_update.configure_parser",
+        ),
+        ("conda_env.cli.main_update.execute", "conda.cli.main_env_update.execute"),
+        ("conda_env.cli.main_vars.var_description", None),
+        ("conda_env.cli.main_vars.var_example", None),
+        ("conda_env.cli.main_vars.list_description", None),
+        ("conda_env.cli.main_vars.list_example", None),
+        ("conda_env.cli.main_vars.set_description", None),
+        ("conda_env.cli.main_vars.set_example", None),
+        ("conda_env.cli.main_vars.unset_description", None),
+        ("conda_env.cli.main_vars.unset_example", None),
+        (
+            "conda_env.cli.main_vars.configure_parser",
+            "conda.cli.main_env_vars.configure_parser",
+        ),
+        (
+            "conda_env.cli.main_vars.execute_list",
+            "conda.cli.main_env_vars.execute_list",
+        ),
+        ("conda_env.cli.main_vars.execute_set", "conda.cli.main_env_vars.execute_set"),
+        (
+            "conda_env.cli.main_vars.execute_unset",
+            "conda.cli.main_env_vars.execute_unset",
+        ),
+        ("conda_env.env.VALID_KEYS", "conda.env.env.VALID_KEYS"),
+        ("conda_env.env.validate_keys", "conda.env.env.validate_keys"),
+        ("conda_env.env.load_from_directory", None),
+        ("conda_env.env.from_environment", "conda.env.env.from_environment"),
+        ("conda_env.env.from_yaml", "conda.env.env.from_yaml"),
+        ("conda_env.env._expand_channels", "conda.env.env._expand_channels"),
+        ("conda_env.env.from_file", "conda.env.env.from_file"),
+        ("conda_env.env.Dependencies", "conda.env.env.Dependencies"),
+        ("conda_env.env.Environment", "conda.env.env.Environment"),
+        ("conda_env.installers.base.ENTRY_POINT", None),
+        (
+            "conda_env.installers.base.InvalidInstaller",
+            "conda.exceptions.InvalidInstaller",
+        ),  # &&&&&&
```

Comment:
?

Reply:
That's a placeholder comment I forgot to remove for a failing test (so I could more easily find it) 😆